react/__tests__/components/__snapshots__/Greeting.test.js.snap

```diff
@@ -19,24 +19,4 @@ exports[`<Greeting /> component should match snapshot with loading 1`] = `
 </DocumentFragment>
 `;
 
-exports[`<Greeting /> component should match snapshot without loading 1`] = `
-<DocumentFragment>
-  <div
-    class="greetingContainer mh4 pv4 t-heading-4 c-on-base nowrap"
-  >
-    <span
-      class="message"
-    >
-      <span>
-        Hello
-      </span>
-      ,
-    </span>
-    <span
-      class="firstName pl2 b"
-    >
-      name
-    </span>
-  </div>
-</DocumentFragment>
-`;
+exports[`<Greeting /> component should match snapshot without loading 1`] = `<DocumentFragment />`;
```

Comment:
This snapshot looks pretty weird. Can you take a look why it is like this?

Reply:
I only typed `yarn test -u`, it was outdated and causing tests to fail...I'll take a look